draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
Even if unverified media isn't allowed, there's an advantage to completing the DTLS handshake early since it requires 2 round trips. An implementation may want to complete the handshake and then buffer or drop incoming media until the fingerprint is verified.